listenbrainz/db/dump_manager.py

```diff
@@ -275,11 +301,39 @@ def write_hashes(location):
             with open(os.path.join(location, '{}.sha256'.format(file)), 'w') as f:
                 sha256sum = subprocess.check_output(['sha256sum', os.path.join(location, file)]).decode('utf-8').split()[0]
                 f.write(sha256sum)
-        except IOError as e:
+        except OSError as e:
             current_app.logger.error('IOError while trying to write hash files for file %s: %s', file, str(e), exc_info=True)
             raise
 
 
+def sanity_check_dumps(location, expected_count):
+    """ Sanity check the generated dumps to ensure that none are empty
```

Comment:
You discarded part of my proposed change, but the method doesn't only check if none of the files are empty, but also check if the expected count matches.